proto/proto/penumbra/view/v1alpha1/view.proto

```diff
@@ -106,6 +106,10 @@ message TransactionPlannerRequest {
     core.crypto.v1alpha1.Fee fee = 2;
     // The memo for the requested TransactionPlan
     string memo = 3;
+    // The valid_before timestamp for the requested TransactionPlan, formatted as an RFC 3339 timestamp
+    optional string valid_before = 4;
```

Comment:
From what I remember of the penumbra architecture, this is  the internal RPC call that converts from UI to internal penumbra data types. As this is more 'user facing', I thought a string would be nicer here.